### DISCUSSION TITLE
ouslazer BRICKED after changing to borderless window mode

### DISCUSSION THREAD

**Discussion by Gluragedon** [ppy/osu#19332] [Q&A] [answered]
I opened osulazer today and wanted to download a beatmap that a friend sent me. I usually download maps using the ingame browser, but today I decided to download it from Chrome and drag and drop them into osu. To do this I changed the window mode to borderless in the settings and the game went black and crashed.
I couldn't reopen the game even after restarting my system (Windows 11).

I've tried a bit of troubleshooting, mainly reinstalling the game, but that didn't help.
At this point, my guess was that my settings and beatmaps are saved separately, but I couldn't find them in file explorer.

I am now out of options and need help to fix osulazer.

Here are a few things you might need to know:
- OS: Windows 11 (Updated)
- osulazer version: not current but i dont know how to look it up without starting the game
- Default Installation path for osulazer
- original osu was installed on this system at one point (is deleted now)
- Dual-Monitor Setup (might have something to do with borderless mode idk

Here are the Logs:
[database.log](https://github.com/ppy/osu/files/9173781/database.log)
[network.log](https://github.com/ppy/osu/files/9173782/network.log)
[performance.log](https://github.com/ppy/osu/files/9173783/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9173784/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9173785/updater.log)

**Comment by peppy** [comment, marked as answer]
Reinstalling will never help, for future reference.

Please provide logs. https://github.com/ppy/osu/discussions/15188

**Reply by Gluragedon** [reply]
I have added the logs now. Hope that helps!

**Reply by peppy** [reply]
The game is not showing an error. This is likely related to your system drivers, or potentially an overlay (such as NVIDIA's game capture app). Please try turning off any overlays and see if it helps.

**Reply by Gluragedon** [reply]
Yes Nvidia's Overlay was the issue thanks for the help!
